### PR TITLE
Fix indentation issue and typos.

### PR DESCRIPTION
**Pull request by keelywan**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -9,21 +9,21 @@
...
-                <h1>Hi, I'm Keely!</h1>
-                <p>I'm a sophomore studying computer science at Cornell University (and exploring economics and 
-                    business). Outisde of school, I'm an avid hiker, recreational tennis player, and huge bookworm. 
-                    Currently re-reading <i>Gone with the Wind</i> - any book recommendations are always welcome! 
-                </p>
-                <p>Check out my <a href="https://www.linkedin.com/in/keely-wan/">LinkedIn</a> or my 
-                <a href="https://www.github.com/keelywan">GitHub</a> for some of my work!</p>
-            </div>
+      <div id="header">
+        <div id="left"><img src="/images/profile.jpg" id="profile"></div>
```
Comment (nickKelman) [inline portfolio/src/main/webapp/index.html]: I would indent this as discussed in the other commit comment history.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -9,21 +9,21 @@
...
+        <div id="right">
+          <h1>Hi, I'm Keely!</h1>
+          <p>I'm a sophomore studying computer science at Cornell University (and exploring economics and
```
Comment (nickKelman) [inline portfolio/src/main/webapp/index.html]: Nit: I think it's generally better styling to move this first paragraph line to a new new line.

Something like:

<p>
  I'm a ....

Comment (nickKelman) [reply portfolio/src/main/webapp/index.html]: Applies to other html tags below.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -9,21 +9,21 @@
...
+            Currently re-reading <i>Gone with the Wind</i> - any book recommendations are always welcome! 
+          </p>
+          <p>Check out my <a href="https://www.linkedin.com/in/keely-wan/">LinkedIn</a> or my 
+            <a href="https://www.github.com/keelywan">GitHub</a> for some of my work!</p>
         </div>
-        <!-- <p>Click here to get a random message:</p>
-        <button onclick="addRandomGreeting()">Hello</button>
-        <div id="greeting-container"></div> -->
+      </div>
+      <!-- <p>Click here to get a random message:</p>
```
Comment (nickKelman) [inline portfolio/src/main/webapp/index.html]: I would delete this commented out code. It's better to keep changes like this that you are considering adding later in local branches that aren't out for review.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -9,21 +9,21 @@
...
+      <button onclick="addRandomGreeting()">Hello</button>
+      <div id="greeting-container"></div> -->
     </div>
-  </body>
+	</body>
```
Comment (nickKelman) [inline portfolio/src/main/webapp/index.html]: This should be unindented.

**Review by nickKelman — COMMENTED** [review]
Thanks for making these changes!

Left a few comments.

**Review by cftorres — APPROVED** [review]

**Review by nickKelman — APPROVED** [review]

**Pull request merged**